build.gradle

```diff
@@ -1,21 +1,53 @@
 plugins {
-    id 'java'
-    id 'org.springframework.boot' version '2.2.7.RELEASE'  apply false
+    id "org.springframework.boot" version "2.4.1" apply false
+    id "io.spring.dependency-management" version "1.0.10.RELEASE" apply false
+    id "org.springdoc.openapi-gradle-plugin" version "1.3.0"
+    id "com.github.johnrengelman.processes" version "0.5.0"
```

Comment:
can you share where this plugin is used?

Reply:
removed